test/testdata/plugin_output.txt

```diff
@@ -0,0 +1,779 @@
+z�
```

Comment:
Gah! They are part of the updates to travis testing, I have removed them from this PR.

It has funny characters because it is actually a serialized proto file - its just that most of its contents are the code that the plugin outputs.